README.md

```diff
@@ -1,41 +1,38 @@
-# Overview
-ESLint configs to enforce VIP's (internal) JS coding standards
+# WordPress VIP ESLint config
 
-# Installation
+ESLint plugin to provide WordPress VIP's (internal) JavaScript and TypeScript coding standards. It is designed to extend [`@wordpress/eslint-plugin`](https://github.com/WordPress/gutenberg/tree/trunk/packages/eslint-plugin), but allows you to choose which preset(s) you want to use.
 
-To add `eslint-config-wpvip` to you project, first install it to `package.json` (not yet published to NPM):
+## Installation
 
-```
-"devDependencies": {
-  ...
-  "eslint-config-wpvip": "github:automattic/eslint-config-wpvip#6b5993b",
-  ...
-}
+Install `eslint` and `eslint-config-wpvip` to your project. Note that this package is not yet published to NPM, so you'll need to grab the latest commit SHA and use an alias.
+
+```sh
+npm install --save-dev eslint eslint-plugin-wpvip@github:Automattic/eslint-config-wpvip#[commit-sha]
 ```
 
-And install the required dependencies:
+If your project uses TypeScript, make sure `typescript` is installed as well.
 
-```
-npm i --save-dev babel-eslint@10 eslint@4 eslint-plugin-flowtype@3 eslint-plugin-import@2 eslint-plugin-jsx-a11y@6 eslint-plugin-react@7 eslint-plugin-wpcalypso@4 eslint-plugin-json@1 eslint-plugin-no-async-foreach
-```
+## Configuration
 
-Then in your `package.json`, add:
+Create an `.eslintrc.json` file with your desired presets. Here is an example that would suit a project that uses React and TypeScript and has Jest unit tests:
 
 ```
-"eslintConfig": {
-  "extends": "wpvip"
+{
+	"extends": [
+		"plugin:@wordpress/eslint-plugin/recommended",
+		"plugin:wpvip/base",
+		"plugin:wpvip/react",
+		"plugin:wpvip/testing",
+		"plugin:wpvip/typescript",
+	]
```

Comment:
Is there anything special that needs to be done in the extending config to use the `cli`-specific settings?